data-model/SDP-sample-records/person_sample1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/person/1",
+  "label": "John bar Me'sam (6th c. CE)",
+  "entity_type": "person",
+  "uniform_name": "John bar Me'sam, Monk, 6th c. CE",
+  "description": "A scribe, operating at the Monastery on Mount Sinai around the 6th century.",
+  "authority_files": [],
+  "associated_dates": [{
+    "type": "floruit",
+    "date_human_readable": "6th c. CE",
+    "date_normalized": "0500/0600"
+  }],
+  "associated_entities": [{
+      "type": "place",
+      "relation": "inhabited",
+      "id": "https://sinaimanuscripts.librar.ucla.edu/place/1",
+      "notes": []
+    }
+  ],
+  "sex": "male",
+  "notes": [],
+  "references": [],
+  "contributor": ["gkessel"]
+}
```

Comment:
it would be good to have a system in place for deprecation and merging. Maybe this just exists in the changelog for the record, but I think having a way to say, 'this person turns out to be the same as this other person, see their record here' would be useful, especially as we might eventually find out that two scribes are actually the same or something like that. I think a record status field and field for redirect IDs would be beneficial.

Reply:
could also include a field for confessional affiliation, if explicitly known. could include ecclesiastic or other rank if desired, e.g. 'monk', 'priest' -- would maybe need a CV that is consistent across languages (e.g., distinguishes Syriac names for priest and monk). We could add other trait-like fields. Some of this depends on how we ingest data from other sources, as I’m not sure we’d want to enter all of this ourselves. But if we have certain kinds of data from, e.g., Syriaca, then would we want to account for how it would ingest?